### ISSUE TITLE
Submit Ecommerce events to Google Analytics

### ISSUE THREAD

**Issue by flo-f** [okgrow/auto-analytics#11]
Hi everyone,

first of all I'd like to thank you for creating and maintaining this great package. It enabled me to implement tracking in very little time.

I'm using `auto-analytics` with Meteor as part of [okgrow:analytics](https://github.com/okgrow/analytics).

Page and standard event tracking works like a charm but I just cannot find a way to make GA recognize my Ecommerce events.

I followed the docs of both _okgrow_ repositories as well as the [segmentio/analytics.js](https://github.com/segmentio/analytics.js) package and its [GA integration](https://github.com/segment-integrations/analytics.js-integration-google-analytics).

Next I tried to follow the [docs for Segment's GA E-Commerce integration](https://segment.com/docs/destinations/google-analytics/#e-commerce).
There, I found two different examples for submitting the `Order Completed` event:

1. [V2 Ecommerce Events: _Order Completed_](https://segment.com/docs/spec/ecommerce/v2/#order-completed)

```javascript
analytics.track('Order Completed', {
 order_id: '...',
 total: 30,
 currency: 'USD',
  products: [ {...} ]
});
```

2. [V1 Ecommerce Events: _Completed Order_](https://segment.com/docs/spec/ecommerce/#completing-an-order)

```javascript
analytics.track('Completed Order', {
  orderId: '...',
  revenue: 25
  currency: 'USD',
  products: [ {...} ]
});
```

I enabled the Ecommerce view in GA but it's not displaying a single cent earned even after trying both tracking events above. The events are submitted to Google Analytics but none of them is recognized as an Ecommerce event by GA.

Am I missing some kind of secret sauce or is this feature just not implemented yet?
If so, are there any plans to add Ecommerce tracking to the `auto-analytics` package?

I hope I didn't overlook any piece of documentation covering this topic. I'd highly appreciate any hint how to get started with Ecommerce tracking.

Best,
Florian

**Comment by cfnelson** [comment]
@flo-f I believe the minified analytics integration is using the `v1` version for Ecommerce Events as the integrations haven't been bumped in awhile. If you are unable to get it to report using the v1 of the `Ecommerce Events`, but you can manually register `.track` events I would suspect that it could be that the minified vendor file we include needs to be updated.

We are currently working on a `v2` of this package where we remove all the integrations and you will have to build them yourself. You can check out the [PR](https://github.com/okgrow/auto-analytics/pull/9), you could clone and switch to the v2 branch and test it. The instructions on how to build the minified analytics.js is [here](https://github.com/okgrow/analytics.js). Let me know if they are not clear as I wrote them up myself and would be happy to get some feedback.

**Comment by flo-f** [comment]
@cfnelson Thank you for the hint! I will look into the `v2` branch but it might take another week until I will find the time. Will submit my feedback ASAP.

Again, thank you very much! Looking forward trying the new version...

**Comment by cfnelson** [comment]
@flo-f Sorry for not alerting you earlier to this but `v1.0.4` was published 2 weeks ago and the vendor file was updated, it seems to reference a `enhancedEcommerce` which might be what you were looking for?

Let me know if `v1.0.4` of this pkg solves your issue 😄 .

**Comment by cfnelson** [comment]
Closing this issue as I believe this issue was resolved since `v1.0.4`. Will be curious to hear if it resolved your issue or if `2.0.0` solved the problem for you. 👍 

Additionally [v2.0.0](https://github.com/okgrow/auto-analytics/releases/tag/v2.0.0) is now released which will allow you to solve the above issue as we no longer bundle any default `analytics.min.js` from segment.io. You must now build your own segment analytics.js bundle and pass it to us. This will allow you to more easily stay up to date with the latest bundles and will decrease the total bundle size significantly.

Check out the Readme for further instructions.
